### ISSUE TITLE
Question

### ISSUE THREAD

**Issue by mikhoul** [Noitidart/Profilist#48]
Hi @Noitidart ,

I was looking to see if the addon will be availlable for webext (FF57) and I see that you seem to have begun the transition. My question is does the beta version webext is working now ?

Best Regards :octocat:

**Comment by Noitidart** [comment]
Thanks @mikhoul for being so in tune. The beta is also a classic/legacy style. I need to transition it to Electron. Which means it will run outside of Firefox, it is no longer possible to make this as a firefox addon. Once its made with electron though, and after users install that, then they can install an addon to connect to the electron. But really it won't do anything other then send messages to the electron app.

**Comment by mikhoul** [comment]
Thanks for the quick reply ! 😃 


Does the button (Firefox addon) will be able to start/close the electron app or the electron app will have to always run in background ?

Regards :octocat:

**Comment by Noitidart** [comment]
Yes it will have to run in the background :( No other way to do this :(

**Comment by mikhoul** [comment]
I've looked in my old doc (before I use an addon) about the profile manager and I found a way to make it easy without an addon to manage the profile (at least under Windows).

You just create a shortcut like this: http://i.imgur.com/07MoP8V.png and it will start directly the profile manager from where you can start and manage/create profiles.

You can also create a shortcut to open directly another profile (without the profile manger) by using the `-no-remote ` switch again. ;-) .

The main advantage with this method don't require to have a program/process running in the background.

I will look later to see how I could link a button/bookmarks to the shortcut on my desktop to start the Profile Manager directly from Firefox for convenience, maybe with a bookmarklet...


Regards :octocat:

**Comment by Noitidart** [comment]
Thank you sir for your initiative to help find workarounds! I also feel disgusting having to run another process in the background. :(

**Comment by ZIEXED** [comment]
Hello Noitidart!
Check this addon, it will help you :)
https://addons.mozilla.org/en-US/firefox/addon/open-with-idm

**Comment by Mafus1** [comment]
There seems to be an addon from Mozilla that does the same: https://testpilot.firefox.com/experiments/containers/
